packages/trader/src/Modules/CFD/Containers/compare-accounts-modal.jsx

```diff
@@ -208,7 +197,31 @@ const getAccounts = ({ landing_companies, platform, is_logged_in }) => {
                 financial_au: localize('50%'),
                 financial_eu: localize('50%'),
                 footnote: localize(
-                    'When the remaining funds in your account is deemed insufficient to cover the leverage or margin requirements, your account will be placed under margin call. To prevent a margin call escalating to a stop out level, you can deposit  additional funds into your account or close any open positions.'
+                    'If your margin level drops below our stop out level, your positions may be closed automatically to protect you from further losses.'
+                ),
+            },
+        },
+        {
+            attribute: localize('Margin call'),
+            mt5: {
```

Comment:
Should we update the values of cards on DMT5 & Deriv X dashboard?

Reply:
No values are changed here, only the footnotes are updated.